资源/定义/flsun_v400.def.json

```diff
@@ -0,0 +1,70 @@
+{
```

Comment:
Please move the printer definition to the `resources/definitions` folder

Reply:
That has been completed as required